main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -37,6 +37,18 @@ object Resolver extends Phase[NamedAst.Root, ResolvedAst.Root] {
     */
   val RecursionLimit: Int = 25
 
+  /**
+    * Symbols of classes that are derivable.
+    */
+  private object DerivableClassSyms {
```

Comment:
I would just have these as private fields. No need to put them inside `DerivableClassSyms` imo. But either is fine.